starter-code/src/viking.js

```diff
@@ -1,11 +1,93 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+
+  receiveDamage(damage) {
+    this.health -= damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+  receiveDamage(damage) {
+    this.health -= damage;
```

Comment:
because the function receiveDamage is already define in Soldier, you could do :
receiveDamage(damage){
   super.receiveDamage(damage)
   ....add your conditions with the sentences
}